desafio-03/leohiroshi/palindromos.py

```diff
@@ -0,0 +1,12 @@
+num01 = int(input('Digite o primeiro valor: '))
+num02 = int(input('Digite o segundo valor: '))
+palindromos = []
+
+for i in range(num01, num02 + 1):
+    if str(i) == str(i)[::-1]: 
```

Comment:
@leohiroshi podemos mandar apenas um desafio for PR, consegue atualizar o PR para deixar apenas o arquivo do D2?